### DISCUSSION TITLE
Minimum duration of `lotus-miner sectors pledge`, 180 or 540 days ?

### DISCUSSION THREAD

**Discussion by nedosa** [filecoin-project/lotus#6463] [Storage Provider] [answered]
Hi,

I'm not sure yet if this is a bug or a plain question, but I've received conflicted answers on the minimum duration for CC sectors. Looking at https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/miner/policy.go#L201 it looks like the minimum pledge duration is 180 days. My questions are 1/ does the `lotus-miner sectors pledge` operation support 180 or 540 days and 2/ is there a way to change the default without recompiling ?

Many thanks for any answers.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
for pledged sectors the min lifetime is 540 days, or close to it - see https://github.com/filecoin-project/filecoin-docs/pull/879

the minimum duration param applies to deals and sector extensions 

@jennijuju can be moved to a discussion - not a lotus issue

**Comment by jennijuju** [comment, marked as answer]
Please see https://docs.filecoin.io/mine/lotus/sector-pledging/#pledging-a-sector

**Reply by nedosa** [reply]
Thanks, appreciate the response, and indeed I should have chosen the Discussions tab, which is more appropriate.
